cat-launcher/src/pages/SettingsPage/components/SettingsForm.tsx

```diff
@@ -0,0 +1,159 @@
+import { zodResolver } from "@hookform/resolvers/zod";
```

Comment:
In SettingsForm.tsx, add a message that "Some settings will be applied on restart." Add a restart now button. Create a restart utility function which uses Tauri API to restart the program.